config/locales/en/layout.yml

```diff
@@ -11,6 +11,7 @@ en:
       sign_out: 'Sign Out'
       admin_panel: 'Administration Panel'
       instance_admin_panel: 'Instance Administration Panel'
+      stop_masquerading: '(Stop Masquerading)'
```

Comment:
![screen shot 2015-12-20 at 17 23 09](https://cloud.githubusercontent.com/assets/4983239/11917353/a881889e-a73e-11e5-9d24-9c13c3ce2202.png)

![screen shot 2015-12-20 at 17 23 33](https://cloud.githubusercontent.com/assets/4983239/11917349/79538acc-a73e-11e5-9e07-410d4c344215.png)

Attached 2 screen shots, think with parentheses would be less confusing ?